homeassistant/components/smartenergy_awattar/config_flow.py

```diff
@@ -0,0 +1,116 @@
+"""Awattar config flow and options flow setup."""
+
+from typing import Any, Literal
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry, ConfigFlow, OptionsFlow
+from homeassistant.const import CONF_SCAN_INTERVAL
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_COUNTRY, CONF_COUNTRY_LIST, DOMAIN
+
+
+def _get_config_schema(default_values: dict) -> vol.Schema:
+    """Define a schema with default values and return it."""
+    return vol.Schema(
+        {
+            vol.Required(
+                CONF_COUNTRY,
+                default=default_values.get(CONF_COUNTRY, CONF_COUNTRY_LIST[0]),
+            ): vol.In(sorted(CONF_COUNTRY_LIST)),
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
+                default=default_values.get(CONF_SCAN_INTERVAL, 10),
+            ): vol.All(vol.Coerce(int), vol.Range(10, 60000)),
+        }
+    )
+
+
+class AwattarConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow for the Awattar component."""
+
+    VERSION: Literal[1] = 1
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlow:
```

Comment:
I'd remove the Optionsflow for now. the shorter this PR is the quicker it will be reviewed by a maintainer.